src/app/shared/components/lib/default/default.module.ts

```diff
@@ -0,0 +1,11 @@
+import { NgModule }           from '@angular/core';
+import { CommonModule }  from '@angular/common';
+
+import { DefaultComponent } from './default.component';
+
+@NgModule({
+  imports:      [ CommonModule ],
+  declarations: [ DefaultComponent ]
+})
```

Comment:
I'm also looking into the code for `rollup-plugin-angular` and will probably use a variant of it to parse the `@Component` and the inline the template and style prior to `ngc`.